<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayerWidget</class>
 <widget class="QWidget" name="MediaPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layoutPlayer">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnPlayPause">
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReset">
         <property name="icon">
          <iconset theme="media-skip-backward"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset theme="media-seek-backward"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForward">
         <property name="icon">
          <iconset theme="media-seek-forward"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="audio"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="slideVolume">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblPosition">
       <property name="text">
        <string notr="true">Position</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblRemaining">
       <property name="text">
        <string notr="true">Remaining</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSlider" name="slideSeek">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnBack</tabstop>
  <tabstop>slideSeek</tabstop>
  <tabstop>btnForward</tabstop>
  <tabstop>slideVolume</tabstop>
  <tabstop>btnPlayPause</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
